app/assets/stylesheets/navigation.scss

```diff
@@ -29,12 +32,35 @@
 }
 #search_bar {
   width: 115px;
+  border-top: 0px;
+  border-right: 0px;
+  border-left: 0px;
+  box-shadow: none;
+  border-color: #ffffff;
```

Comment:
Color literals like `#ffffff` should only be used in variable declarations; they should be referred to via variable everywhere else.